### DISCUSSION TITLE
Is there an equivalent to Schema tab in GraphiQL2

### DISCUSSION THREAD

**Discussion by bilalebi** [graphql/graphiql#3167] [Q&A / Support]
In GraphQL playground, there is a schema tab (near `Docs`) that allows me to view the schema, I couldn't find one when I switched to GraphiQL2, am I missing something? If not are there any plans to implement it in the future?

<img src="https://github.com/graphql/graphiql/assets/62993769/f64a8730-088f-4430-87b2-045e0bb5a644" width="40" height="150" />

**Comment by LekoArts** [comment]
It's the first icon in the sidebar, the "Documentation explorer"

**Reply by bilalebi** [reply]
Thank you for taking the time to answer my question, as far as I understood, "Documentation explorer" to me is the equivalent of `Docs` tab in the playground.

Schema on the other hand lays down the whole graphql schema as you can [in this example](https://barcelona-urban-mobility-graphql-api.netlify.app/graphql).

The closed feature I found so far is the [`Explorer` plugin](https://github.com/OneGraph/graphiql-explorer).

**Reply by LekoArts** [reply]
Ah, I see 👍 Yeah, I don't think this exists. You can use https://github.com/graphql/graphiql/tree/main/packages/graphiql-plugin-explorer but if you need the "Schema" tab then that would be a new plugin that needs to be created.
